functional_requirements/requirements.txt

```diff
@@ -1,111 +1,188 @@
 
 - button (A)
 - button (B)
 
-The device shall power on when button A is pressed for more than 3 seconds
-The initial display state upon power-on shall be clock
+REQ 1.2 Display
 
-The clock shall deactivate when buttons A+B are pressed for more than 3 seconds
+The device shall communicate with the user thought the output display.
 
-At power-on the clock shall read the settings and load the clock time from mobile phone. If connection is not available, default 00:00 is used.
+REQ 1.3 Activation
 
+The device shall power on when button A is pressed for more than 3 seconds.
 
-Display - views
+REQ 1.3 Deactivation
 
-- Clock with hands: default view at start-up
+The clock shall power off when buttons A+B are pressed for more than 3 seconds in any moment.
 
-The watch shall display the current time using clock with hands display (hours, minutes and seconds hands)
-The watch shall change to "Clock with digits" when button A is pressed.
-The watch shall change to "Change display view" view when button B is pressed.
 
-- Clock with digits: 
+
+PART 2 Display views
+
+REQ 2.1 views
+
+The views provided by the watch shall be the following ones:
+
+- Clock view
+- Display Selection view
+- Step Count view
+- Hearth rate view
+- ECG view
+- Sp02 view
+- Map view
+- Setting view
+
+REQ 2.2 Initial view
+
+The initial display state upon power-on shall be the clock view.
+
+REQ 2.3 Clock view
 
 The watch shall display the current time using clock with digits (hours and minutes)
 The watch shall display the battery life indicator
-The watch shall change to "Clock with hands" when button A is pressed.
-The watch shall change to "Change display view" view when button B is pressed.
-
+The watch shall change to "Display Selection view" view when button B is pressed.
 
-- Change display view:
+REQ 2.4 Display Selection view
 
 Display options are: 
-	News
-	Weather
+
+	Clock
+	Step count
 	Heart rate
 	ECG
 	Sp02
 	Map
 	Settings
 	Power off
 	
+Default option is "Clock"	
 The watch shall change selected view when button A is pressed.
 The watch shall activate the selected view when button B is pressed. 
 The watch shall deactivate when "Power off" is selected.
 
+REQ 2.5 Step Count view
 
-- News view:
-
-The watch shall connect to the mobile phone to retrieve the news info through the bluetooth connection. If connection is available, the news page is displayed otherwise an error message is displayed.
-The watch shall change to "Change display view" view when button B is pressed.
-
-- Weather view:
-
-The watch shall connect to the mobile phone to retrieve the weather info through the bluetooth connection. If connection is available, the weather page is displayed otherwise an error message is displayed.
+The watch shall display the current step count. If not available, an error message is displayed.
 The watch shall change to "Change display view" view when button B is pressed.

```

Comment:
Instead of "Change display view" can be say "Display Selection view" ? 

This is what is mentioned in REQ 2.1 Views

Reply:
fixed